registry/remote/repository.go

```diff
@@ -1423,22 +1431,17 @@ func (s *manifestStore) indexReferrersForPush(ctx context.Context, desc ocispec.
 		return nil
 	}
 
-	ok, err := s.repo.pingReferrers(ctx)
```

Comment:
Since `pingReferrers` is now only used in the delete path, we can remove the `zeroDigest` hack.

Reply:
Found the old converstation: https://github.com/opencontainers/distribution-spec/issues/359#issuecomment-1294902270

This design was for performance consideration. We used the digest of a non-existing manifest to avoid server to query the referrers.